internal/pfcpsim/server.go

```diff
@@ -184,57 +199,51 @@ func (P pfcpSimService) CreateSession(ctx context.Context, request *pb.CreateSes
 				AddQERID(downlinkAppQerID).
 				WithFARID(downlinkFarID).
 				MarkAsDownlink().
-				BuildPDR(),
-		}
+				BuildPDR()
 
-		fars := []*ieLib.IE{
-			// UplinkFAR
-			session.NewFARBuilder().
-				WithID(uplinkFarID).
+			pdrs = append(pdrs, uplinkPDR)
+			pdrs = append(pdrs, downlinkPDR)
+
+			uplinkFAR := session.NewFARBuilder().
+				WithID(uint32(ID)).
 				WithAction(session.ActionForward).
 				WithDstInterface(ieLib.DstInterfaceCore).
 				WithMethod(session.Create).
-				BuildFAR(),
+				BuildFAR()
 
-			// DownlinkFAR
-			session.NewFARBuilder().
-				WithID(downlinkFarID).
+			downlinkFAR := session.NewFARBuilder().
+				WithID(uint32(ID + 1)).
 				WithAction(session.ActionDrop).
 				WithMethod(session.Create).
 				WithDstInterface(ieLib.DstInterfaceAccess).
 				WithZeroBasedOuterHeaderCreation().
-				BuildFAR(),
-		}
+				BuildFAR()
 
-		qers := []*ieLib.IE{
-			// TODO make rates configurable by pfcpctl
-			// session QER
-			session.NewQERBuilder().
-				WithID(sessQerID).
-				WithMethod(session.Create).
-				WithUplinkMBR(60000).
-				WithDownlinkMBR(60000).
-				Build(),
+			fars = append(fars, uplinkFAR)
+			fars = append(fars, downlinkFAR)
 
-			// Uplink application QER
-			session.NewQERBuilder().
+			uplinkAppQER := session.NewQERBuilder().
 				WithID(uplinkAppQerID).
 				WithMethod(session.Create).
 				WithQFI(qfi).
 				WithUplinkMBR(50000).
 				WithDownlinkMBR(30000).
 				WithGateStatus(gateStatus).
-				Build(),
+				Build()
 
-			// Downlink application QER
-			session.NewQERBuilder().
+			downlinkAppQER := session.NewQERBuilder().
 				WithID(downlinkAppQerID).
 				WithMethod(session.Create).
 				WithQFI(qfi).
 				WithUplinkMBR(50000).
 				WithDownlinkMBR(30000).
 				WithGateStatus(gateStatus).
-				Build(),
+				Build()
+
+			qers = append(qers, uplinkAppQER)
+			qers = append(qers, downlinkAppQER)
+
+			ID++
```

Comment:
Since we use 2 IDs (1 for UL and 1 for DL), we should increase by 2.
```suggestion
			ID+=2
```